fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/visitor/ExpressionVisitor.java

```diff
@@ -433,6 +434,10 @@ public R visitWindowFrame(WindowFrame windowFrame, C context) {
         return visit(windowFrame, context);
     }
 
+    public R visitMatch(Match match, C context) {
+        return visit(match, context);
+    }
```

Comment:
u should add all visitXXX here, not only add the base class. and all other visitXX should call `visitMatch`

Reply:
I will update it